README.md

```diff
@@ -1 +1,215 @@
-# unicode-characters
+---
+datapackage:
+  title: Unicode Codes
+  description: The Unicode Consortium is the standards body for the internationalization of software and services. Deployed on more than 20 billion devices around the world, Unicode also provides the solution for internationalization and the architecture to support localization.
+  created: 2024-01-01
+  updated: 2024-01-31
+  licenses:
+  - path: http://opendatacommons.org/licenses/pddl/
+    title: Open Data Commons Public Domain Dedication and License v1.0
+  sources:
+  - path: https://home.unicode.org/
+    title: Link to data source
+  resources:
+  - name: unicode-codes
+    title: Unicode codes
+    description: Unicode codes
+    lastModified: 2024-10-10
+    path: data.csv
+---
+
+
+The following table describes the format and meaning of each field in a data entry in the UnicodeData. Fields which contain normative information are so indicated.
+
+| Field | Name | Status | Explanation |

```

Comment:
Are these field information? If so, could we add it as table schema in the datapackage.json please?